Dashboard/templates/index.html

```diff
@@ -21,140 +21,77 @@
             </div>
+            {% endfor %}
         </div>
 
         <h4>Completed Scans</h4>
         <div class="list">
+            {% for item in completed %}
             <div class="indexRow row">
                 <div class="col-sm">
-                    <h6>Scan #1</h6>
-                    <h6>Submitted: 00:00:00</h6>
+                    <h6>Scan #{{ item["scanNo"] }}</h6>
+                    <h6>Submitted: {{ item["submitTime"] }}</h6>
                 </div>
                 <div class="col-sm">
-                    <h6>Size: 4GB</h6>
-                    <h6>Number of files: 1000</h6>
+                    <h6>Size: {{ item['size'] }}</h6>
+                    <h6>Number of files: {{ item['numFiles'] }}</h6>
                 </div>
                 <div class="col-sm-5">
                     <div class="progress">
-                        <div class="progress-bar" style="width: 100%;"></div>
-                    </div>
-                </div>
-            </div>
-            <div class="indexRow row">
-                <div class="col-sm">
-                    <h6>Scan #1</h6>
-                    <h6>Submitted: 00:00:00</h6>
-                </div>
-                <div class="col-sm">
-                    <h6>Size: 4GB</h6>
-                    <h6>Number of files: 1000</h6>
-                </div>
-                <div class="col-sm-5">
-                    <div class="progress">
-                        <div class="progress-bar" style="width: 100%;"></div>
-                    </div>
-                </div>
-            </div>
-            <div class="indexRow row">
-                <div class="col-sm">
-                    <h6>Scan #1</h6>
-                    <h6>Submitted: 00:00:00</h6>
-                </div>
-                <div class="col-sm">
-                    <h6>Size: 4GB</h6>
-                    <h6>Number of files: 1000</h6>
-                </div>
-                <div class="col-sm-5">
-                    <div class="progress">
-                        <div class="progress-bar" style="width: 100%;"></div>
-                    </div>
-                </div>
-            </div>
-            <div class="indexRow row">
-                <div class="col-sm">
-                    <h6>Scan #1</h6>
-                    <h6>Submitted: 00:00:00</h6>
-                </div>
-                <div class="col-sm">
-                    <h6>Size: 4GB</h6>
-                    <h6>Number of files: 1000</h6>
-                </div>
-                <div class="col-sm-5">
-                    <div class="progress">
-                        <div class="progress-bar" style="width: 100%;"></div>
+                        <div class="progress-bar" style="width: {{ item['stats'] }}%;"></div>
                     </div>
                 </div>
             </div>
+            {% endfor %}
         </div>
     </div>
+
+    <script>
+        function openUploader() {
+            window.location = "http://localhost:5000/newScan";
```

Comment:
@maanas-talwar if we can get the base URL from the env variables it would be better.

Reply:
Sure